src/test/java/StringClassTest.java

```diff
@@ -0,0 +1,84 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+public class StringClassTest {
+
+    @Nested
+    class 요구_사항_1 {
+        @Test
+        void split_했을_때_1과_2로_분리() {
+            //given
+            String test = "1,2";
+
+            //when
+            String[] result = test.split(",");
+
+            //then
+            assertThat(result).contains("1", "2");
+            assertThat(result).containsExactly("1", "2");
```

Comment:
```suggestion
            String[] result = "1,2".split(",");
            assertThat(result).contains("1", "2");
            assertThat(result).containsExactly("1", "2");
```

위와 같이 구현하는 것은 어떨까?
//given, when, then을 사용할 경우 오히려 테스트 코드 라인 수가 길어지면서 가독성이 떨어지지 않을까?